web/components/admin/ApplicantInfo/ShortField/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ShortField extends React.Component {
+  render() {
+    const { label, value, isUrl } = this.props;
+    return (
+      <div className="pad-sides-xs pad-ends-xs">
+        <div className="applicant-info-label pad-bottom-m">{label}</div>
+        <div className="applicant-short-info">
+          {isUrl ? (<div className="link"><a href={value}>{value}</a></div>) : value}
```

Comment:
1. If you click this it will open the URL in the same tab. Could you make this open up in a new tab?
2. In our constraints we only verify that the resume link is a proper URL. Unfortunately this means that for the other link fields, people may have entered values like `linkedin.com/x` or `www.github.com/x`. Anything without an `http://` or `https://` (specifically the `//`) will be considered a relative URL. So for the linkedin example, the URL would resolve to `nwhacks.io/#/application/assessment/linkedin.com/x`). You should be able to fix this by prefixing all URLs that don't start with `http` with `//` (ex: `href={value.startsWith('http') ? value : '//' + value}`)

Reply:
Good catch!